docs/resources/server.md

```diff
@@ -145,6 +153,7 @@ resource "ionoscloud_server" "example" {
 - `ssh_key_path` - (Optional)[list] List of paths to files containing a public SSH key that will be injected into IonosCloud provided Linux images.  Also accepts ssh keys directly. Required for IonosCloud Linux images. Required if `image_password` is not provided.
 - `image_password` - (Optional)[string] Required if `ssh_key_path` is not provided.
 - `type` - (Optional)[string] Server usages: [ENTERPRISE](https://docs.ionos.com/cloud/compute-engine/virtual-servers/virtual-servers) or [CUBE](https://docs.ionos.com/cloud/compute-engine/virtual-servers/cloud-cubes). This property is immutable.
+- `label` - (Optional) A label can be seen as an object with only two required fields: `key` and `value`, both of the `string` type. Please check the example presented above to see how a `label` can be used in the plan.
```

Comment:
[Optional][list]

Reply:
@cristiGuranIonos I intentionally omitted to add this because it is not really a basic list. An example of basic list would be `ssh_key_path` for which we write this in the terraform plan:
```
ssh_key_path = ["value1", "value2", ...]
```
For these labels we use a different approach so I choose to omit the type and instead give some details related to the structure of a label. However, for more clarity, I changed the documentation a little bit. Please take another look and tell me if it looks okay.